simple/wso2ei.sh

```diff
@@ -0,0 +1,3067 @@
+#!/bin/bash
+
+#-------------------------------------------------------------------------------
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#--------------------------------------------------------------------------------
+
+
+set -e
+# bash variables
+k8s_obj_file="deployment.yaml"; str_sec=""
+WSO2_SUBSCRIPTION_USERNAME=''
+WSO2_SUBSCRIPTION_PASSWORD=''
+NODE_IP=''
+
+#bash functions
+function usage(){
+  echo "Usage: "
+  echo -e "-d, --deploy     Deploy WSO2 Enterprise Integrator"
+  echo -e "-u, --undeploy   Undeploy WSO2 Enterprise Integrator"
+  echo -e "-h, --help       Display usage instrusctions"
+}
+function undeploy(){
```

Comment:
Some functions have a new line space before and some don't . Shall we fix this?

Reply:
ok. what is the best practice here? To have a newline or not?

Reply:
Best practice is to have an empty line between functions, mainly for the purpose of readability.